tests/qadence/test_matrices.py

```diff
@@ -470,6 +470,36 @@ def test_projector_tensor(projector: AbstractBlock, exp_projector_mat: Tensor) -
     assert torch.allclose(projector_mat, exp_projector_mat, atol=1.0e-4)
```

Comment:
I guess this `atol` could be as low as `ATOL_64` no?

Reply:
Nope, this is the best I could achieve for this test.